usaspending_api/awards/v2/lookups/matview_lookups.py

```diff
@@ -1,6 +1,7 @@
 default_mapping = {
     "Recipient Name": "recipient_name",
     "Recipient DUNS Number": "recipient_unique_id",
+    "recipient_id": "_recipient_id",  # recipient_id conflicts with another column in the model
```

Comment:
Should the key here not be `Recipient Id` to match the consistency of the other keys?

Reply:
You'd think.  It's not for display.  The only other not-for-display field is `internal_id`.  I don't feel strongly either way.  Consistency with the current endpoint or consistency with the rest of the app.  You tell me.

Reply:
I guess I prefer consistency with `internal_id`. At least there is a reason for it being the way it is, all I care about.